CRM/Atomfeeds/Page/Feed.php

```diff
@@ -149,7 +138,7 @@ public function _getFeed($url) {
         $blog[] = $item;
       }
       if ($blog) {
-        CRM_Core_BAO_Cache::setItem($blog, 'dashboard', 'blog');
+        Civi::cache('dashboard')->set('blog', $blog, 86400);
```

Comment:
In the original code, the constant `CRM_Atomfeeds_Page_Feed::CACHE_DAYS` was used to calculate the number of seconds for the expiration period. I notice you have it now set to a fixed period of 24 hours.
I would multiply 86400 by `CRM_Atomfeeds_Page_Feed::CACHE_DAYS`, or otherwise remove the CACHE_DAYS constant from  `CRM_Atomfeeds_Page_Feed`.